modules/nodes-nodes-working-deleting.adoc

```diff
@@ -6,60 +6,59 @@
 [id="nodes-nodes-working-deleting_{context}"]
 = Deleting nodes from a cluster
 
-When you delete a node using the CLI, the node object is deleted in Kubernetes,
-but the pods that exist on the node are not deleted. Any bare pods not
-backed by a replication controller become inaccessible to {product-title}.
-Pods backed by replication controllers are rescheduled to other available
-nodes. You must delete local manifest pods.
+To delete a node from the {product-title} cluster, scale down the appropriate `MachineSet` object.
 
-.Procedure
+[IMPORTANT]
+====
+When a cluster is integrated with a cloud provider, you must delete the corresponding machine to delete a node. Do not try to use the `oc delete node` command for this task.
+====
```

Comment:
This block is the new content. 

The other changes are rearrangement and reformatting, but should also be checked.